qiskit/result/mitigation/local_readout_mitigator.py

```diff
@@ -0,0 +1,306 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+"""
+Readout mitigator class based on the 1-qubit local tensored mitigation method
+"""
+
+
+from typing import Optional, List, Tuple, Iterable, Callable, Union
+import numpy as np
+
+from qiskit.providers.backend import Backend
+from qiskit.providers.models.backendproperties import BackendProperties
```

Comment:
This might be the source of your aer import errors, Either use a string for the type hint, drop the type hint (my preference), or use a type checking if statement for these imports: https://docs.python.org/3/library/typing.html#typing.TYPE_CHECKING